src/modules/note/services/note-received.service.ts

```diff
@@ -23,33 +69,33 @@ export class NoteReceivedService {
   async handleNoteReceived(payload: NoteReceivedWebhookPayload) {
     const {
       object_attributes: note,
-      merge_request: mergeRequest,
       project_id: projectId,
       user: author,
     } = payload;
 
-    const discussion = await this.gitlab.MergeRequestDiscussions.show(
+    const mergeRequest = await this.gitlab.MergeRequests.show(
       projectId,
-      mergeRequest.iid,
-      //@ts-expect-error error
-      note.discussion_id,
+      payload.merge_request.iid,
     );
 
+    const assigneesUsernames = mergeRequest.assignees.map(
+      (assignee) => assignee.username,
+    ) as string[];
+    const reviewersUsernames = mergeRequest.reviewers.map(
+      (reviewer) => reviewer.username,
+    ) as string[];
```

Comment:
Why do you need to type this explicitly ?

Reply:
This is unknown type

Reply:
arf